### DISCUSSION TITLE
Waystone compatibility

### DISCUSSION THREAD

**Discussion by efficient-efi** [seymourimadeit/respawn-save-points#2] [General]
Thanks for the great mod!
It would be useful to have the inventory saved in waystones as well(I play mcver 1.20.1 as most players)

**Comment by seymourimadeit** [comment]
Converted into a discussion simply because it better fit here: 

I will probably work on something relating to allowing other block entities support this system, but that's a long way to go since I have to focus on a lot of real life stuff right now.

**Reply by efficient-efi** [reply]
Yeah, your real life is the most important thing.
